Frameworks/AVFoundation/AVAudioPlayer.mm

```diff
@@ -65,25 +70,25 @@ @implementation AVAudioPlayer {
 
 - (instancetype)init {
     if (self = [super init]) {
-        _mediaElement = [WXCMediaElement make];
-        _mediaElement.autoPlay = NO;
-        _mediaElement.volume = 1.0f;
-        _hiddenView = [[_UIHiddenMediaView alloc] initWithFrame:{ 0, 0, 0, 0 } xamlElement:_mediaElement];
+        _mediaElement = Controls::MediaElement();
+        _mediaElement.AutoPlay(false);
+        _mediaElement.Volume(1.0f);
+        _hiddenView = [[_UIHiddenMediaView alloc] initWithFrame:{ 0, 0, 0, 0 } xamlElement:objcwinrt::to_rtobj(_mediaElement)];
 
-        _lastState = _mediaElement.currentState;
+        _lastState = _mediaElement.CurrentState();
 
         // Set up events
         __weak AVAudioPlayer* weakSelf = self;
 
-        [_mediaElement addCurrentStateChangedEvent:^(RTObject* sender, WXRoutedEventArgs* e) {
+        _mediaElement.CurrentStateChanged([weakSelf] (auto&& sender, auto&& e) {
```

Comment:
>auto&& sender, auto&& e [](start = 54, length = 23)

would prefer types on these...across a lot of these files.  our CCs say that we scope the use of auto to cases where declaring the type would be redundant.